scripts/exportSitesAsCsv.js

```diff
@@ -40,6 +41,21 @@ function consolidateOnSiteId(sites) {
   });
 }
 
+function writeCSV(sites, fileName) {
+  const fields = ['id', 'github', 'domain', 'users'];
```

Comment:
To futureproof (and match the functionality of the prior library), can you get the fields dynamically rather than hardcoding

Reply:
Yes. I had that and removed it for declarative simplicity but I'm happy to go the other way.

Reply:
Dynamic field implementation pushed. 

The `json-to-csv` library [used](https://www.npmjs.com/package/json-to-csv?activeTab=code) the Ramda library, effectively calling `Ramda.keys(sites[0])` to determine the fields. I'm using `Object.keys(sites[0])` to avoid that dependency. And unlike the old library I'm doing it inside the try/catch block in case `[]` or `keys()` raise an error.